app/api/v2/handlers/fact_source_api.py

```diff
@@ -22,43 +22,76 @@ def add_routes(self, app: web.Application):
         router.add_put('/sources/{id}', self.create_or_update_source)
         router.add_delete('/sources/{id}', self.delete_source)
 
-    @aiohttp_apispec.docs(tags=['sources'])
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Retrieve all fact sources.',
+                          description='Returns a list of all fact sources, including custom-created ones.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(SourceSchema(many=True, partial=True))
     async def get_fact_sources(self, request: web.Request):
         sources = await self.get_all_objects(request)
         return web.json_response(sources)
 
-    @aiohttp_apispec.docs(tags=['sources'])
+    @aiohttp_apispec.docs(tags=['sources'],
+                          summary='Retrieve a fact source by its id.',
+                          description='Returns a fact source, given a source id.',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'description': 'The id of the fact source',
+                              'schema': {'type': 'string'},
+                              'required': 'true'
+                            }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
     @aiohttp_apispec.response_schema(SourceSchema(partial=True))
```

Comment:
Ben had pointed out in my PR that the response_schemas should also have a description, like:
` @aiohttp_apispec.response_schema(SourceSchema(partial=True), description='...')`

Other than that looks good to me!

Reply:
Thanks!